docs/proposals/00-dbaas-community-catalog.md

```diff
@@ -0,0 +1,370 @@
+    - [Version Skew Strategy](#version-skew-strategy)
+  - [Production Readiness Review Questionnaire](#production-readiness-review-questionnaire)
+    - [Monitoring Requirements](#monitoring-requirements)
+    - [Dependencies](#dependencies)
+    - [Troubleshooting](#troubleshooting)
+  - [Implementation History](#implementation-history)
+  - [Drawbacks](#drawbacks)
+  - [Alternatives](#alternatives)
+<!-- /toc -->
+
+## Summary
+
+<!--
+This section is incredibly important for producing high-quality, user-focused
+documentation such as release notes or a development roadmap. It should be
+possible to collect this information before implementation begins, in order to
+avoid requiring implementors to split their attention between writing release
+notes and implementing the feature itself. KEP editors and SIG Docs
+should help to ensure that the tone and content of the `Summary` section is
+useful for a wide audience.
+
+A good summary is probably at least a paragraph in length.
+
+Both in this section and below, follow the guidelines of the [documentation
+style guide]. In particular, wrap lines to a reasonable length, to make it
+easier for reviewers to cite specific portions, and to minimize diff churn on
+updates.
+
+[documentation style guide]: https://github.com/kubernetes/community/blob/master/contributors/guide/style-guide.md
+-->
+OLM Catalogs are part of Operator Lifecycle Manager ([OLM](https://olm.operatorframework.io/)) framework to provide set of operators that extend Kubernetes. DBaaS Catalog would provide set of operators to extend Kubernetes to deliver DBaaS Platform based on it.
+
+## Motivation
+
+<!--
+This section is for explicitly listing the motivation, goals, and non-goals of
+this KEP.  Describe why the change is important and the benefits to users. The
+motivation section can optionally provide links to [experience reports] to
+demonstrate the interest in a KEP within the wider Kubernetes community.
+
+[experience reports]: https://github.com/golang/go/wiki/ExperienceReports
+-->
+The Operator Lifecycle Manager ([OLM](https://olm.operatorframework.io/)) extends Kubernetes to provide a declarative way to install, manage, and upgrade Operators on a cluster. [OLM Catalogs](https://olm.operatorframework.io/docs/concepts/crds/catalogsource/) is the way to distribute set of operator that can be installed onto a cluster via OLM.
+
+There are some catalogs already available:
+  - [OperatorHub.io](https://operatorhub.io): https://github.com/k8s-operatorhub/community-operators
+  - [RH Catalogs](https://docs.openshift.com/container-platform/4.6/operators/understanding/olm-rh-catalogs.html#olm-rh-catalogs_olm-rh-catalogs): https://github.com/redhat-openshift-ecosystem/community-operators-prod
+
+Existing catalogs have all possible operators and could be overkill for most of the systems, as well as for developers to ensure that some sub-set of it works good together to solve particular problems.
```

Comment:
```suggestion
Existing catalogs include all available operators. This could be overkill for most of the systems and for developers to ensure that some sub-set of it works well together to solve particular problems.
```